### PR TITLE
Bugfix: SMA pairing reliability

### PR DESCRIPTION
**Pull request by dalathegreat**
### What
This PR improves the success rate of getting a SMA battery paired

### Why
..it took pairing time down from ~45 mins and a 20% chance of it working to ~15 minutes and a 70% chance of it working the first time.

### How
We now look at the enable line status (via the inverter allowes closing contactors), and when this is ON, we periodically broadcast messages every 100ms

### CODE REVIEW

File: Software/src/inverter/BYD-SMA.cpp [hunk 1]
```diff
@@ -254,15 +256,17 @@ void receive_can_inverter(CAN_frame rx_frame) {
...
-    previousMillis60s = currentMillis;
-    transmit_can(&SMA_158, can_config.inverter);
-    transmit_can(&SMA_358, can_config.inverter);
-    transmit_can(&SMA_3D8, can_config.inverter);
-    transmit_can(&SMA_458, can_config.inverter);
-    transmit_can(&SMA_518, can_config.inverter);
-    transmit_can(&SMA_4D8, can_config.inverter);
+  // Send CAN Message every 100ms if we're enabled
+  if (datalayer.system.status.inverter_allows_contactor_closing) {
+    if (currentMillis - previousMillis60s >= 100) {
```
Comment (lenvm) [inline Software/src/inverter/BYD-SMA.cpp:261]: Shouldn’t `previousMillis60s` be `previousMillis100s`?

File: Software/src/inverter/BYD-SMA.cpp [hunk 2]
```diff
@@ -254,15 +256,17 @@ void receive_can_inverter(CAN_frame rx_frame) {
...
+      previousMillis60s = currentMillis;
```
Comment (lenvm) [inline Software/src/inverter/BYD-SMA.cpp:262]: same comment here

Comment (dalathegreat) [reply Software/src/inverter/BYD-SMA.cpp]: Fixed in [6d32ce1](https://github.com/dalathegreat/Battery-Emulator/pull/452/commits/6d32ce187c6511580814d8b41642bcfd8c52dd55)

File: Software/src/inverter/BYD-SMA.cpp (hunk 1 above)
Comment (dalathegreat) [reply Software/src/inverter/BYD-SMA.cpp]: Yes! fixed in  [6d32ce1](https://github.com/dalathegreat/Battery-Emulator/pull/452/commits/6d32ce187c6511580814d8b41642bcfd8c52dd55)

**Pull request merged**